src/pymovements/events/events.py

```diff
@@ -287,6 +287,70 @@ def event_property_columns(self) -> list[str]:
         event_property_columns -= set(self._additional_columns)
         return list(event_property_columns)
 
+    def _filter_by_prefix(self, prefix: str) -> pl.DataFrame:
```

Comment:
why does this need to be private -- I think it would be a great function in general for the user? familiarity with polars is low so this could be an excellent way of prevent users to learn polars syntax.